apps/cowswap-frontend/src/modules/appData/updater/AppDataHooksUpdater.ts

```diff
@@ -55,7 +55,8 @@ export function AppDataHooksUpdater(): null {
     )
 
     // Permit data is not loaded yet, wait until it's loaded
-    if (permitData === undefined) {
+    // Permit data with sc-wallets is always undefined
+    if (permitData === undefined && !isSmartContractWallet) {
```

Comment:
I don't get the comment.
It's saying it's always undefined with SC wallets, but the check is for NOT SC wallets.

Reply:
Added more description here